parm/config/config.gefs

```diff
@@ -0,0 +1,11 @@
+#!/bin/ksh -x
```

Comment:
Which job is sourcing this file?
It may be needed in subsequent PR's.
Consider adding when it is used.

Reply:
Currently, this config file is just for placeholder. If there is no need for now until we need this file, I can delete this file.